src/map/script.c

```diff
@@ -13516,72 +13516,76 @@ static BUILDIN(getmapinfo)
-			case MF_NIGHTENABLED:       script_pushint(st,map->list[m].flag.nightenabled); break;
-			case MF_NOBASEEXP:          script_pushint(st,map->list[m].flag.nobaseexp); break;
-			case MF_NOJOBEXP:           script_pushint(st,map->list[m].flag.nojobexp); break;
-			case MF_NOMOBLOOT:          script_pushint(st,map->list[m].flag.nomobloot); break;
-			case MF_NOMVPLOOT:          script_pushint(st,map->list[m].flag.nomvploot); break;
-			case MF_NORETURN:           script_pushint(st,map->list[m].flag.noreturn); break;
-			case MF_NOWARPTO:           script_pushint(st,map->list[m].flag.nowarpto); break;
-			case MF_NIGHTMAREDROP:      script_pushint(st,map->list[m].flag.pvp_nightmaredrop); break;
-			case MF_NOCOMMAND:          script_pushint(st,map->list[m].nocommand); break;
-			case MF_NODROP:             script_pushint(st,map->list[m].flag.nodrop); break;
-			case MF_JEXP:               script_pushint(st,map->list[m].jexp); break;
-			case MF_BEXP:               script_pushint(st,map->list[m].bexp); break;
-			case MF_NOVENDING:          script_pushint(st,map->list[m].flag.novending); break;
-			case MF_LOADEVENT:          script_pushint(st,map->list[m].flag.loadevent); break;
-			case MF_NOCHAT:             script_pushint(st,map->list[m].flag.nochat); break;
-			case MF_NOEXPPENALTY:       script_pushint(st,map->list[m].flag.noexppenalty ); break;
-			case MF_GUILDLOCK:          script_pushint(st,map->list[m].flag.guildlock); break;
-			case MF_TOWN:               script_pushint(st,map->list[m].flag.town); break;
-			case MF_AUTOTRADE:          script_pushint(st,map->list[m].flag.autotrade); break;
-			case MF_ALLOWKS:            script_pushint(st,map->list[m].flag.allowks); break;
-			case MF_MONSTER_NOTELEPORT: script_pushint(st,map->list[m].flag.monster_noteleport); break;
-			case MF_PVP_NOCALCRANK:     script_pushint(st,map->list[m].flag.pvp_nocalcrank); break;
-			case MF_BATTLEGROUND:       script_pushint(st,map->list[m].flag.battleground); break;
-			case MF_RESET:              script_pushint(st,map->list[m].flag.reset); break;
-			case MF_NOTOMB:             script_pushint(st,map->list[m].flag.notomb); break;
-			case MF_NOCASHSHOP:         script_pushint(st,map->list[m].flag.nocashshop); break;
-			case MF_NOAUTOLOOT:         script_pushint(st, map->list[m].flag.noautoloot); break;
-			case MF_NOVIEWID:           script_pushint(st, map->list[m].flag.noviewid); break;
-			case MF_PAIRSHIP_STARTABLE: script_pushint(st, map->list[m].flag.pairship_startable); break;
-			case MF_PAIRSHIP_ENDABLE:   script_pushint(st, map->list[m].flag.pairship_endable); break;
-			case MF_NOSTORAGE:          script_pushint(st, map->list[m].flag.nostorage); break;
-			case MF_NOGSTORAGE:         script_pushint(st, map->list[m].flag.nogstorage); break;
+	if (m >= 0) {
+		switch (i) {
+			case MF_NOMEMO:                  script_pushint(st, map->list[m].flag.nomemo); break;
```

Comment:
Fix indentation. Cases should have same indentation level as `switch`. (See [style guide](https://github.com/HerculesWS/Hercules/wiki/Coding-Style#chapter-1-indentation).)

Reply:
updated